cli/src/main/java/org/apache/iotdb/cli/Cli.java

```diff
@@ -18,6 +18,8 @@
  */
 package org.apache.iotdb.cli;
 
+import static org.apache.iotdb.cli.utils.IoTPrinter.println;
+
```

Comment:
Is this needed?

Reply:
Yes. Otherwise, I need to change "println" to "IoTPrinter.println" in Cli.java.

Reply:
> Is this needed?

Yes. Otherwise, I need to change "println" to "IoTPrinter.println" in many places.

Reply:
> > Is this needed?
> 
> Yes. Otherwise, I need to change "println" to "IoTPrinter.println" in many places.

I think it is ok to use `IoTPrinter.println` ....